static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -156,34 +142,14 @@ function BreadcrumbItem({
         ))}
 
         {isErrorFrame(frame) || isFeedbackFrame(frame) ? (
-          <CrumbErrorIssue frame={frame as ErrorFrame} />
```

Comment:
Since the code just checked that frame is one of these types, we shouldn't re-cast it again. 

using the typescript `as` keyword is a code smell.